Userland/Libraries/LibM/math.cpp

```diff
@@ -1130,6 +1130,37 @@ float nearbyintf(float value) NOEXCEPT
 {
     return internal_to_integer(value, RoundingMode { fegetround() });
 }
+
+double fdim(double x, double y) NOEXCEPT
```

Comment:
Your implementation doesn't pay attention to overflow.

> If x- y is positive and overflows, a range error shall occur and fdim(), fdimf(), and fdiml() shall return the value of the macro HUGE_VAL, HUGE_VALF, and HUGE_VALL, respectively.

>If the correct value would cause underflow, a range error may occur, and fdim(), fdimf(), and fdiml() shall return 
> (if the IEC 60559 Floating-Point option is not supported) an implementation-defined value no greater in magnitude than DBL_MIN, FLT_MIN, and LDBL_MIN, respectively.

Reply:
Yes, you are right, it doesn't handle overflow. I'll probably have to calculate the difference and then check with the isinf macro if it overflows so errno can be set to ERANGE